### DISCUSSION TITLE
Placed components do not show up in Design view

### DISCUSSION THREAD

**Discussion by Zac06** [eclipse/windowbuilder#974] [Q&A] [answered]
I'm using Eclipse 2024-12 with the latest WindowBuilder version, on Arch Linux using KDE Plasma 6.2.4. I have tried reinstalling all the plugins, themes, changing GTK and system themes, however all the placed components on the Design view do not show up. They do show up when compiling or viewing preview though.
Here is a picture describing the issue:

![immagine](https://github.com/user-attachments/assets/2c51b566-3496-41f6-b1fe-38bea353a49c)

**Comment by ptziegler** [comment, marked as answer]
Based on the screenshot, you seem to be using Wayland which is, as noted in https://github.com/eclipse-windowbuilder/windowbuilder/issues/633, not supported at this time.

As a workaround, I would suggest starting the Eclipse IDE with following environment variable and value: `GDK_BACKEND=x11`

**Reply by Zac06** [reply]
Thank you so much.
